Funcky/Monads/Result.cs

```diff
@@ -22,24 +23,29 @@ private Result(Exception error)
             _error = error;
         }
 
+        [Pure]
         public static bool operator ==(Result<TValidResult> lhs, Result<TValidResult> rhs)
             => lhs.Equals(rhs);
 
+        [Pure]
         public static bool operator !=(Result<TValidResult> lhs, Result<TValidResult> rhs)
             => !lhs.Equals(rhs);
 
+        [Pure]
```

Comment:
This should not be pure

Reply:
Fixed in 1fcfc1d9f6b835884e914ba49399ff5ff436c258